framework/po/Collection.js

```diff
@@ -5,6 +5,18 @@ class Collection extends AbstractComponent {
         super(locator);
         this.components.set(name, locator);
     }
+    /**
+    * Set new collection by selector or collection object
+    * @param {*} name - collection name
+    * @param {*} selector - can be selector of collection or collection object
+    */
+    defineCollection(name, selector) {
```

Comment:
if we have this both on Collection and Component we can describe it in AbstractComponent class